client/styles/base/_icons-svg.scss

```diff
@@ -1,10 +1,19 @@
+.icon-default-speaker-conference-avatar {
```

Comment:
Not sure what you mean about change - assets are hard to change by definition, you just replace them.